pom.xml

```diff
@@ -126,5 +128,79 @@
         <hadoop.version>${hadoop3.version}</hadoop.version>
       </properties>
     </profile>
+    <profile>
+      <!-- default HBase profile -->
+      <id>hbase2</id>
+      <activation>
+        <property>
+          <name>!hbase.profile</name>
+        </property>
+      </activation>
+      <properties>
+        <hbase.version>${hbase2.version}</hbase.version>
+      </properties>
+      <!-- For testing against ZK -->
+      <dependencies>
+	<dependency>
+	  <groupId>org.apache.hbase</groupId>
+	  <artifactId>hbase-zookeeper</artifactId>
+	  <version>${hbase2.version}</version>
+	  <scope>test</scope>
+	</dependency>
+	<dependency>
+	  <groupId>org.apache.hbase</groupId>
+	  <artifactId>hbase-zookeeper</artifactId>
+	  <version>${hbase2.version}</version>
+	  <type>test-jar</type>
+	  <scope>test</scope>
+	</dependency>
+      </dependencies>
+    </profile>
+    <profile>
+      <id>hbase1</id>
+      <activation>
+        <property>
+          <name>hbase.profile</name>
+          <value>1</value>
+        </property>
+      </activation>
+      <properties>
+        <hbase.version>${hbase1.version}</hbase.version>
+      </properties>
+      <dependencies>
+	<dependency>
+          <groupId>org.apache.hbase.thirdparty</groupId>
+          <artifactId>hbase-shaded-miscellaneous</artifactId>
+          <version>${hbase-thirdparty.version}</version>
+	</dependency>
+	<!-- For testing against ZK -->
+	<dependency>
+	  <groupId>org.apache.hbase</groupId>
+	  <artifactId>hbase-testing-util</artifactId>
+	  <version>${hbase1.version}</version>
+	  <scope>test</scope>
+	</dependency>
+	<dependency>
+	  <groupId>org.apache.hbase</groupId>
+	  <artifactId>hbase-testing-util</artifactId>
+	  <version>${hbase1.version}</version>
+	  <type>test-jar</type>
+	  <scope>test</scope>
+	</dependency>
+	<dependency>
+	  <groupId>org.apache.hadoop</groupId>
+	  <artifactId>hadoop-minikdc</artifactId>
+	  <version>${hadoop.version}</version>
+	  <exclusions>
+	    <exclusion>
```

Comment:
Exclusion needed here to avoid a dependency resolution problem with -Dhadoop.version=2.9.2